client/components/remove-button/style.scss

```diff
@@ -0,0 +1,109 @@
+// ==========================================================================
+// Buttons
+// ==========================================================================
+
+.button-remove {
+	color: darken( $gray, 10% );;
+	cursor: pointer;
+	display: inline-block;
+	margin: 0;
+	outline: 0;
+	overflow: hidden;
+	font-weight: 600;
+	text-overflow: ellipsis;
+	text-decoration: none;
+	vertical-align: top;
+	box-sizing: border-box;
+	font-size: 14px;
+	line-height: 21px;
+	border-radius: 4px;
+	padding: 7px 14px 9px 10px;
+	-webkit-appearance: none;
+	appearance: none;
+
+	&:hover {
+		border-color: lighten( $gray, 10% );
+		color: $gray-dark;
+	}
+	&:active {
+		border-width: 2px 1px 1px;
+	}
+	&:visited {
+		color: $gray-dark;
+	}
+	&[disabled],
+	&:disabled {
+		color: lighten( $gray, 30% );
+		background: $white;
+		cursor: default;
+
+		&:active {
+			border-width: 1px 1px 2px;
+		}
+	}
+	&:focus {
+		border-color: $blue-medium;
+		box-shadow: 0 0 0 2px $blue-light;
+	}
+	&.hidden {
+		display: none;
+	}
+  &.is-compact {
```

Comment:
@ebinnion My editor must be screwed up, I could use some help fixing that because it doesn't look that way to me.